src/themes/default/pages/Product.vue

```diff
@@ -141,19 +141,26 @@
             <div class="row m0 mb15" v-if="product.type_id !== 'grouped' && product.type_id !== 'bundle'">
               <div>
                 <label class="qty-label flex" for="quantity">{{ $t('Quantity') }}</label>
-                <input
+                <base-input
                   type="number"
-                  min="0"
-                  class="m0 no-outline qty-input py10 brdr-cl-primary bg-cl-transparent h4"
+                  :min="1"
+                  class="m0 no-outline qty-input"
                   id="quantity"
                   focus
+                  @input="validateQty"
```

Comment:
```suggestion
                  @input="this.$v.$touch()"
```
Let's follow the convention we have all over the project